cmd/api.go

```diff
@@ -139,6 +139,11 @@ func generateCollage(request *CollageRequest) (image.Image, error) {
 		fontSize = 2
 	}
 
+	// set DisplayAlbum to false if the method isn't for albums
+	if request.Method == "artist" || request.Method == "track" {
```

Comment:
Hmm I'm not sure this is the best place for this check, since we'll end up with a lot of if statements for customising each option. Might be better to just set this to `false` in `generateCollageForArtist` and `generateCollageForTrack`.

Alternatively, I was just planning to only display it if it was non-empty when we actually write the image, in the function `placeText`.

Reply:
Yeah that sounds good. Keep in mind, users may force `album` to be `true` despite the UI restrictions, we just have to  account for that so they don't get a gap in the text. Should I close this PR, and leave it to the other PRs?